docs/advanced.md

```diff
@@ -0,0 +1,142 @@
+## Advanced usage
+
+In the [quickstart](./quickstart.md), we have created a simple user model and ran a few queries.
+However, Prisma and the Go client are designed to work with relations between models.
+
+We already created a post model, such as for a blog. Let's assume we want to add a user model, and connect these models
```

Comment:
Any reason you went with User instead of Comments?

Reply:
Ah, no. I kinda just wrote what I could think of first. Do you think using comments is better?

Reply:
Probably fine. The reason I suggested `Comment` is that it naturally extends your schema without needing to adjust the data of what you've already done. 

If you create a post, then you add a User model, you need to set the user_id on the Post schema. This will need to be nullable if you've already already added posts. Just more steps.

Reply:
Ah, that's true. I will change it to comments then.